src/standalone/Vegetation/component_models.jl

```diff
@@ -132,30 +132,9 @@ end
 """
      set_canopy_prescribed_field!(component::AbstractCanopyComponent,
                                   p,
-                                  t0,
+                                  t,
                                  ) end
 
-Sets the spatially and temporally varying prescribed fields of the `component`
-with their initial values.
-
-These fields are stored in the aux-state and *should not* depend on the prognostic
-state `Y` or other diagnostic variables stored in `p`; this allows them
-to be updated first, prior to updating the rest of the aux state and prognostic state.
-
-However, there is no
-guarantee on the order of operations in terms of when diagnostic auxiliary
-variables are updated vs. prescribed field auxiliary variables; any required
-order of operations must be enforced by the developer who writes the update_aux
-function.
```

Comment:
Should we keep this docstring?

Reply:
I cleaned it up a bit, but I think it still holds, so keeping. let me know if you had something in mind for not including it!